x-pack/platform/plugins/private/upgrade_assistant/server/lib/es_deprecations_status/migrations.ts

```diff
@@ -269,7 +269,53 @@ export const getEnrichedDeprecations = async (
   esClient: ElasticsearchClient,
   log: Logger
 ): Promise<EnrichedDeprecationInfo[]> => {
-  const esDeprecations = (await esClient.migration.deprecations()) as EsDeprecations;
+  let esDeprecations = (await esClient.migration.deprecations()) as EsDeprecations;
+  esDeprecations = {
+    ...esDeprecations,
+    cluster_settings: [
+      {
+        level: 'warning',
+        message:
+          'Template patterns are no longer using `template` field, but `index_patterns` instead',
```

Comment:
```suggestion
          'Template patterns now use the `index_patterns` fields instead of the `template` fields',
```
I reckon we can swap the two fields to lead with the one that's in use in the favour of the one that's not (also avoiding the negative "no longer doing X").